content/blog/2022/11/airgap.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Installing and Running Kubewarden In Air-Gapped Environments
+authors:
+- Raul Cabello Martin
+date: 2022-11-28
+---
+
+We are glad to announce that deploying Kubewarden in air-gapped environments has been simplified and documented! For that, you will need a 
+private OCI registry accessible by your Kubernetes cluster. Kubewarden policies are WebAssembly modules; therefore
+they can be stored inside an OCI-compliant registry as OCI artifacts. 
+For an air gap installation you need to download all the Kubewarden container images and policies in your workstation, 
+then move them to your private OCI registry. Check the Kubewarden [docs](https://docs.kubewarden.io/operator-manual/airgap/install) 
+for a step-by-step guide.
+
+We have added new commands to `kwctl` [v1.3.1](https://github.com/kubewarden/kwctl/releases/tag/v1.3.1) in order to simplify air gap deployment: 
+- `save`: saves a list of Kubewarden policies provided as input in a tar.gz file.
+- `load`: loads Kubewarden policies from a tar.gz file into an OCI registry.
+
+There are helpers' scripts that make easier the air gap installation. You can find them in our [utils repository](https://github.com/kubewarden/utils/tree/main/scripts):
```

Comment:
Not a native English speaker, shouldn't the sentence begin in a different way:

```suggestion
There are helper scripts that make easier the air gap installation. You can find them in our [utils repository](https://github.com/kubewarden/utils/tree/main/scripts):
```

Reply:
not sure about this one. This came from divya's comment https://github.com/kubewarden/kubewarden.io/pull/153/commits/58b917ae78d39d337127cdd407f29f348067c06f

Reply:
I did not know if we had multiple helpers or just a single helper. If we have multiple helpers then the current one is correct. If not, what Flavio suggested is correct.

Reply:
we have multiple, then it is correct as it is now.